azure/ipa.tf

```diff
@@ -211,7 +211,7 @@ storage:
     enabled: false
   indicoStorageClass:
     enabled: false
-    name: "default"
+    name: "azurefile"
```

Comment:
I had also used these mountOptions @ashmuck - from the file I sent you
```
  mountOptions:
  - dir_mode=0777
  - file_mode=0777
  - uid=0
  - gid=0
  - nobrl
  - nosharesock
```
which are what msft recommends in their docs

Reply:
(I assume uid and gid being 0 vs 1000 is not the important factor here)

Reply:
Added